### PR TITLE
[stable3.0] chore(CI): Adjust testing matrix for Nextcloud 26

### PR DESCRIPTION
**Pull request by GretaD**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         php-versions: ['8.0', '8.1', '8.2']
-        nextcloud-versions: ['master']
+        nextcloud-versions: ['master', 'stable26']
```
Comment (ChristophWurst) [inline .github/workflows/test.yml:13]:
```suggestion
        nextcloud-versions: ['stable26']
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
master is 27 and doesn't work. Mail stable3.0 can only test against server stable26

**Review by ChristophWurst — APPROVED** [review]

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         php-versions: ['8.0', '8.1', '8.2']
-        nextcloud-versions: ['master']
+        nextcloud-versions: ['stable26']
```
Comment (ChristophWurst) [inline .github/workflows/test.yml:13]: there are two more usages of `master` in this workflow file

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Comment by ChristophWurst** [comment]
psalm worlkflow should also switch to dev-stable26

**Review by ChristophWurst — APPROVED** [review]

**Review by st3iny — APPROVED** [review]

**Pull request merged**